### DISCUSSION TITLE
obsidian 是可以用一辈子不换的笔记软件吗

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#43] [主发布站的评论系统]
# obsidian 是可以用一辈子不换的笔记软件吗

可以访问此文的双链版本，获得完整阅读体验： obsidian 是可以用一辈子不换的笔记软件吗 - 🌲 oldwinterの数字花园
 我是哪根葱  从近 10 年经历看，我算得上是一个『少数派』：  高光🔦：高考全省排名前 1% 的 [[🐤小镇做题家]] 摆烂🕹：大学 89 人的专业绩点倒数第 1 的垫底小胖 高光🔦：突击 3 个月考研成功，被幸运女神临幸 摆烂🕹：读研没发 paper 也没去实习，硕士文凭生产流水线上的不合格产品 高光🔦：IT 类大厂连续 2 年获得最高绩效，日均工作 14 小时，人称 [[内卷の王]] 摆烂🕹：换了个非常轻松的工作，工作时长和收入都只有原来一半的低欲望 [[躺平一族]]   另外，我还是一个全面拥抱数字化的『少数派』：  打字⌨️：非文字工作者，搜狗输入统计打了 400 万字，击败 99% 阅读📖：除了考研，其他学习都只看电子版，多看 + 微信读书共计 2000+ 小时阅读时长，击败 99% 数码📱：安卓主流厂商系统都玩过，苹果全家桶也都玩过，root，越狱都门清儿   标题只是希望能增加点进来的人的数量，蹭了一下这个知乎问答的热度 「我来 wolai」是可以用了一辈子不用换的笔记软件吗？ - 知乎 (zhihu.

https://garden.oldwinter.top/obsidian-%E6%98%AF%E5%8F%AF%E4%BB%A5%E7%94%A8%E4%B8%80%E8%BE%88%E5%AD%90%E4%B8%8D%E6%8D%A2%E7%9A%84%E7%AC%94%E8%AE%B0%E8%BD%AF%E4%BB%B6%E5%90%97

**Comment by Mikeal-red** [comment]
方便问下作者大大mac端的工作流或者笔记工作流嘛，闪念和卡片笔记是放在哪里的

**Reply by oldwinter** [reply]
dailynote放闪念笔记。库里的所有md文件，都可以理解为广义的卡片笔记。最近看到一篇记笔记好文章，推荐给你：https://sspai.com/post/77144

**Reply by Mikeal-red** [reply]
哇，居然这么快就回复了，感谢
